packages/core/helper-plugin/MIGRATION_GUIDE.md

```diff
@@ -17,6 +17,20 @@ import { DateTimePicker } from '@strapi/helper-plugin';
 import { DateTimePicker } from '@strapi/design-system';
 ```
 
+### Form
+
+This component aliased `Formik`, something we're working towards removing. The `Form` component and it's sibling exports from `@strapi/strapi/admin` should be used instead:
+
+```ts
+// Before
+import { Form } from '@strapi/helper-plugin';
+
+// After
+import { Form } from '@strapi/strapi/admin';
+```
+
+Users should note that any use of the Formik library will no longer work & insted should look at the documentation for the `Form` component.
```

Comment:
should this be mentioned in the breaking changes list https://www.notion.so/strapi/831ef9cf0d66425a9f97d4981f96c1cc?v=2ee276ead71b41adb17ef477a47c96ac ?

Reply:
I'll probably write a section on the helper-plugin being removed and link to this migration guide but yes